cmd/nerdctl/build.go

```diff
@@ -75,7 +75,7 @@ func newBuildCommand() *cobra.Command {
 }
 
 func getBuildkitHost(cmd *cobra.Command) (string, error) {
-	if cmd.Flags().Changed("buildkit-host") {
+	if cmd.Flags().Changed("buildkit-host") || os.Getenv("BUILDKIT_HOST") != "" {
```

Comment:
IMHO, The piece of code here is may belong to the https://github.com/containerd/nerdctl/pull/932

Maybe we got some rebase issue here?